src/js/iov/player.js

```diff
@@ -37,8 +37,13 @@ export default function (iov) {
 
 
     self._fault = function(err) {
-        //TODO: Change the video poster to a failure image
-        console.log(err);
+        _ThePlayer.errors.extend({
```

Comment:
Can this be put in initialization logic rather than extending the errors object every time there is a fault?